kdumpctl

```diff
@@ -1729,8 +1758,27 @@ main()
 			reset_crashkernel_for_installed_kernel "$2"
 		fi
 		;;
+	test)
+		is_kernel_loaded "$DEFAULT_DUMP_MODE"
+		if [[ "$?" -ne 0 ]]; then
+			derror "Kdump should be operational before test."
+			exit 1
+		fi
+		read -p "DANGER!!! Will perform a kdump test by crashing the system, proceed? (Y/N): " input
+		case $input in
+		[Yy] )
+			dinfo "Start kdump test..."
+			;;
+		* )
+			dinfo "Operation cancelled."
+			exit 0
+			;;
+		esac
+		set_vmcore_creation_status 'clear' "$VMCORE_CREATION_STATUS"
+		echo c > /proc/sysrq-trigger
```

Comment:
I'd prefer if you moved this into a separate function rather than having it all in the switch case.

Reply:
OK agreed, I have the same thought when coding...

Reply:
Thanks